contracts/MerkleDistributor.sol

```diff
@@ -0,0 +1,115 @@
+pragma solidity 0.6.4;
+
+import "./interface/IBEP20.sol";
+import "./interface/ITokenHub.sol";
+import "./interface/IMerkleDistributor.sol";
+import "./lib/SafeMath.sol";
+import "./System.sol";
+import "./MerkleProof.sol";
+
+contract MerkleDistributor is IMerkleDistributor, System {
```

Comment:
`MerkleDistributor` seems not a good name,  as it is not only  Merkle verification.

Reply:
rename to `AirDrop`